connect/api/src/main/java/org/apache/kafka/connect/source/SourceConnector.java

```diff
@@ -28,4 +30,39 @@
     protected SourceConnectorContext context() {
         return (SourceConnectorContext) context;
     }
+
+    /**
+     * Signals whether the connector supports exactly-once delivery guarantees with a proposed configuration.
+     * Developers can assume that worker-level exactly-once support is enabled when this method is invoked.
+     * For backwards compatibility, the default implementation will return {@code null}, but connector developers are
```

Comment:
Nit: using a new paragraph makes this stand out more.
```suggestion
     * <p>For backwards compatibility, the default implementation will return {@code null}, but connector developers are
```

Reply:
Ack, done. (And added a few more paragraph breaks in other places too).